evm/src/keccak/registers.rs

```diff
@@ -0,0 +1,252 @@
+use crate::keccak::keccak_stark::NUM_ROUNDS;
+
+/// A register which is set to 1 if we are in the `i`th round, otherwise 0.
+pub(crate) const fn reg_step(i: usize) -> usize {
+    debug_assert!(i < NUM_ROUNDS);
+    i
+}
+
+const R: [[u8; 5]; 5] = [
+    [0, 36, 3, 41, 18],
+    [1, 44, 10, 45, 2],
+    [62, 6, 43, 15, 61],
+    [28, 55, 25, 21, 56],
+    [27, 20, 39, 8, 14],
+];
+
+const RC: [u64; 24] = [
+    0x0000000000000001,
+    0x0000000000008082,
+    0x800000000000808A,
+    0x8000000080008000,
+    0x000000000000808B,
+    0x0000000080000001,
+    0x8000000080008081,
+    0x8000000000008009,
+    0x000000000000008A,
+    0x0000000000000088,
+    0x0000000080008009,
+    0x000000008000000A,
+    0x000000008000808B,
+    0x800000000000008B,
+    0x8000000000008089,
+    0x8000000000008003,
+    0x8000000000008002,
+    0x8000000000000080,
+    0x000000000000800A,
+    0x800000008000000A,
+    0x8000000080008081,
+    0x8000000000008080,
+    0x0000000080000001,
+    0x8000000080008008,
+];
+
+const RC_BITS: [[u8; 64]; 24] = [
```

Comment:
Would you mind moving these into a `constants` module? Would prefer if we keep `registers` just for register indices

Reply:
Ah yes, good idea!